includes/wikia/models/NavigationModel.class.php

```diff
@@ -82,6 +82,15 @@ public function clearMemc( $key = self::WIKIA_GLOBAL_VARIABLE ) {
 		WikiaDataAccess::cachePurge(
 			$this->getMemcKey( $key )
 		);
+
+		// Purging mcache set by invocation of NavigationModel::getTree() by NavigationModel::getLocalNavigationTree();
```

Comment:
this method is not used at all, and I wonder if we should still keep it or just remove